packages/edit-site/src/components/sidebar-navigation-screen-navigation-menu/index.js

```diff
@@ -74,6 +116,14 @@ export default function SidebarNavigationScreenNavigationMenu() {
 
 	return (
 		<SidebarNavigationScreenWrapper
+			actions={
+				<ScreenNavigationMoreMenu
+					menuTitle={ decodeEntities( menuTitle ) }
+					handleDelete={ handleDelete }
+					handleSave={ handleSave }
+					handleDuplicate={ handleDuplicate }
```

Comment:
```suggestion
					onDelete={ handleDelete }
					onSave={ handleSave }
					onDuplicate={ handleDuplicate }
```

Let's use a consistent `on` prefix. We're passing a prop that the component should call when something happens. We are not passing a handler.

Reply:
✅